example/README.md

```diff
@@ -0,0 +1,171 @@
+	```
+
+2. Search curves by meta data.
+
+	For this use case define meta criteria for DK1 area wind power production.
+
+	```python
+	>>> meta_criteria = {'commodity': 'POW',
+	                     'category': ['WND', 'Intraday'], 
+	                     'unit': 'MWH/H',
+	                     'area': 'DK1', 
+	                     'data_type': ['A', 'F']}
+	>>> curves = session.search(**meta_criteria)
+	['pro dk1 wnd intraday ec00da mwh/h cet min15 f',
+	 'pro dk1 wnd intraday lastec mwh/h cet min15 f',
+	 'pro dk1 wnd intraday tso mwh/h cet min15 f',
+	 'pro dk1 wnd intraday mwh/h cet min15 a',
+	 'pro dk1 wnd ec00 mwh/h cet min15 f',
+	 'pro dk1 wnd ec12 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs00 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs06 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs12 mwh/h cet min15 f',
+	 'pro dk1 wnd gfs18 mwh/h cet min15 f',
+	 'pro dk1 wnd ec00ens mwh/h cet min15 f',
+	 'pro dk1 wnd ec12ens mwh/h cet min15 f',
+	 'pro dk1 wnd mwh/h cet min15 a']
+	```
+
+3. For the first example, filter on intraday curves and retrieve data for the selected curves.
+
+	```python
+	>>> intraday_curves = [c for c in curves if 'INTRADAY' in c.categories]
+	>>> data = {}
+	>>> label = {}
+	>>> issue_date = pytz.timezone('CET').localize(
+						datetime.strptime('2018-04-12 00:00', "%Y-%m-%d %H:%M"))
+	>>> date_from = issue_date + timedelta(minutes=15)
+	>>> date_to = date_from + timedelta(hours=24)
+	>>> for c in intraday_curves:
+	>>>     if c.curve_type == 'INSTANCES':
+	>>>         ts = c.get_instance(issue_date)
+	>>>         if ts is not None:
+	>>>             label[c.name] = ';'.join(c.sources)
+	>>>             data[c.name] = ts.to_pandas()
+	>>>     elif c.curve_type == 'TIME_SERIES':
+	>>>         data[c.name] = c.get_data(data_from=date_from, 
+	>>>                                   data_to=date_to).to_pandas()
+	>>>         label[c.name] = 'Actual'
+	```
+
+4.  Plot selected data
+
+	```python
+	>>> fig = tls.make_subplots(len(data), cols=1, shared_xaxes=True)
+	>>> for key in data.keys():
+	>>>     ts = data[key]
+	>>>     fig.append_trace({'x': ts.index, 'y': ts.values, 'type': 'scatter', 
+	>>>                       'name': label[key]}, 1, 1)
+	>>> title = ' '.join([c.commodity,
+	>>>                   ' '.join(c.categories),
+	>>>                   c.area,
+	>>>                   issue_date.strftime("%Y-%m-%d %H:%M"),
+	>>>                   c.time_zone])
+	>>> fig['layout'].update(title=title)
+	>>> fig['layout'].update(yaxis=dict(title=c.unit))    
+	>>> offline.iplot(fig)
+	```
+
+	![Chart](chart1.png)
+
+5.  Retrive data for more instances and build contract development series.
+
+	```python
+	>>> curve = intraday_curves[0]
+	>>> latest_instance = curve.get_latest(with_data=False)
+	>>> issue_date_str = latest_instance.issue_date
+	>>> issue_date = latest_instance.tz.localize(
+						datetime.strptime(latest_instance.issue_date, 
+ 	                                      "%Y-%m-%dT%H:%M:%SZ"))
```

Comment:
Again does this need to be so complex?

Reply:
that's the format of issue_date string returned from get_latest

Reply:
Should the issue_date element be a string when it is returned?  Or should it be a datetime object?  Even if parsing here, the string format is standardized so you should not have supply the format string.